POC/ColorPicker/ColorPicker/TransparentWindow.xaml

```diff
@@ -9,6 +9,20 @@
         WindowStyle="None"
         ShowInTaskbar="False"
         Background="#01000000"
+        Loaded="OnLoad"
         MouseDown="OnMouseDown"
-        KeyDown="OnKeyDown">
+        KeyDown="OnKeyDown"
+        MouseMove="OnMouseMove">
+    <Window.Resources>
+        <SolidColorBrush x:Key="MagnifiedImageBorderColor" Color="#FF808080"/>
+        <SolidColorBrush x:Key="MagnifiedImagePixelBorderColor" Color="#FFFF0000"/>
+    </Window.Resources>
+    <Canvas>
+        <Border Name="MouseFollower" BorderBrush="{StaticResource MagnifiedImageBorderColor}" BorderThickness="2" CornerRadius="2" Visibility="Hidden">
```

Comment:
This line's a little long, you might want to wrap it (same with the rectangle below). I don't know what your team's conventions are though 😊

Reply:
Will do!